### DISCUSSION TITLE
SFML 3

### DISCUSSION THREAD

**Discussion by 3lyrion** [mikke89/RmlUi#714] [General]
SFML 3 was released, and stencil test was added there. It would be nice to update the backend files.

**Comment by mikke89** [comment]
Yes, I've been working on adding support for SFML 3 lately. It seems SDL 3 is also around the corner, so I am working on that too while I'm at it.

**Reply by mikke89** [reply]
Added now: https://github.com/mikke89/RmlUi/commit/b261d161a5649cee346341d5e67ae380098e9400, in addition to SDL 3 support :)

**Reply by 3lyrion** [reply]
Thank you!
